app/helpers/orchestration/orchestrator.rb

```diff
@@ -0,0 +1,168 @@
+# frozen_string_literal: true
+
+# Orchestrator: Methods for managing active playbook context
+module Orchestration::Orchestrator # rubocop:todo Metrics/ModuleLength
+  include Orchestration::Playbooks
+
+  available = Orchestration::Playbooks.constants.reject { |m| m == :Templates }
+  modules = available.map { |m| ('Orchestration::Playbooks::' + m.to_s).constantize }
+
+  playbooks = {}
+  modules.each { |m| playbooks[m::NAME] = m::PLAYBOOK }
+
+  PLAYBOOKS = playbooks.freeze
+
+  def workflow_configuration(playbook, workflow, option)
+    # For now, assume that if no workflow (ie it is nil, it is a general option)
+    base_configuration = workflow.nil? ? PLAYBOOKS.dig(playbook, :general, :base, option) : PLAYBOOKS.dig(playbook, :workflows, workflow, :base, option)
+    custom_options = workflow.nil? ? PLAYBOOKS.dig(playbook, :general, :custom_options,
+                                                   option) : PLAYBOOKS.dig(playbook, :workflows, workflow, :custom_options, option)
+
+    return if base_configuration.nil?
```

Comment:
Isn't the line below going to handle this case?

```suggestion
```

Reply:
@ChristineDuong

Reply:
This line is just a protection against if for some reason a configuration is not listed in the `base_configuration` (within the template) but is present within the playbook. I believe otherwise it will error within the `nested_configuration` function.

Reply:
ok that makes sense, thanks!